File: Ejercicio3/Coche.java
package Ejercicio3;
//Crea una clase Coche con marca, modelo y velocidad
public class Coche {
    public String marca;
    public String modelo;
    public int velocidad;

    public Coche(String marca, String modelo, int velocidad) {
        this.marca=marca;
        this.modelo=modelo;
        this.velocidad=velocidad;
    }
    //a) Agrega un método acelerar () que aumente la velocidad en 10
    public void acelerar() {
        velocidad += 10;
    }
    //b) Agrega un método frenar () que disminuya la velocidad en 5
    public void frenar() {
        velocidad -= 5;
    }
    public static void main(String[] args) {
        //c) Crea dos coches, aceléralos, frénalos y muestra sus velocidades
        Coche c1=new Coche("Toyota","Kia",50);
        Coche c2=new Coche("Ford","Mustang",80);
        c1.acelerar();
        c2.acelerar();
        c1.acelerar();
        c2.frenar();
        c1.frenar();
         // mostrar velocidades 
         System.out.println("coche 1: " + c1.velocidad);
         System.out.println("coche 2: " + c2.velocidad);
     
    }
}

File: Poo/Ejercicio3/coche.py
class Coche:
    # Crea una clase Coche con marca, modelo y velocidad
    def __init__(self, marca, modelo, velocidad):
        self.marca = marca
        self.modelo = modelo
        self.velocidad = velocidad

    # a) Método para acelerar que aumenta la velocidad en 10
    def acelerar(self):
        self.velocidad += 10

    # b) Método para frenar que disminuye la velocidad en 5
    def frenar(self):
        self.velocidad -= 5


if __name__ == "__main__":
    # c) Crea dos coches, aceléralos, frénalos y muestra sus velocidades
    c1 = Coche("Toyota", "Kia", 50)
    c2 = Coche("Ford", "Mustang", 80)

    c1.acelerar()
    c2.acelerar()
    c1.acelerar()
    c2.frenar()
    c1.frenar()

    # Mostrar velocidades
    print(f"Coche 1: {c1.velocidad} km/h")
    print(f"Coche 2: {c2.velocidad} km/h")